src/noir/common/log.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <fc/log/appender.hpp>
+#include <fc/log/log_message.hpp>
+#include <fc/log/logger_config.hpp>
+
+namespace noir::log {
+
+  class tmlog_appender : public fc::appender {
+  public:
+    explicit tmlog_appender( const fc::variant& args );
```

Comment:
We don't put whitespace around arguments.

```suggestion
    explicit tmlog_appender(const fc::variant& args);
```